src/current/cockroachcloud/authorization.md

```diff
@@ -35,66 +35,124 @@ For the main pages covering users and roles at the SQL level within a specific d
+
+### Billing Coordinator
+
+Users with this role in an organization can [manage billing for that organization](billing-management.html) through the {{ site.data.products.db }} console billing page at [`https://cockroachlabs.cloud/billing/overview`](https://cockroachlabs.cloud/billing/overview).
 
-Cluster Developers can view the details of clusters and can change their IP allowlist configuration. This role can be granted for specific clusters or for all clusters in the organization.
+Note that billing can also be managed by the [Org Administrator (legacy) role](#org-administrator-legacy).
+
+### Cluster Operator
+
+This role can be granted for one or more specific clusters, or for all clusters in the organization. It allows users and service accounts to perform a variety of cluster functions:
+
+- *Users* with this role can perform the following *console operations*:
+
+	- View a cluster's [Overview page](cluster-overview-page.html), which displays its configuration, attributes and statistics, including cloud provider, region topography, and available and maximum storage and request units.
+	- Manage a cluster's databases from the [Databases Page](databases-page.html).
+	- [Scale a cluster's nodes](cluster-management.html#scale-your-cluster)
+	- View and configure a cluster's authorized networks from the [Networking Page](network-authorization.html).
+	- View backups in a cluster's [Backups Page](use-managed-service-backups.html#backups-page)
+	- [Restore a cluster](use-managed-service-backups.html#restore-a-cluster) to a backup state.
+	- View a cluster's Jobs from the [Jobs page](jobs-page.html)
+	- View a cluster's Metrics from the [Metrics page](metrics-page.html)
+	- View a cluster's Insights from the [Insights page](insights-page.html)
+	- [Upgrade](upgrade-to-v23.1.html#step-5-start-the-upgrade) a cluster's CRDB version.
+	- View a cluster's [PCI-readiness status (Dedicated Advanced clusters only)](cluster-overview-page.html?filters=dedicated#pci-ready-dedicated-advanced).
+	- Send a test alert from the [alert's page](alerts-page.html).
+	- Configure single sign-on (SSO) enforcement.
+
+- *Service accounts* with this role can perform the following *API operations*:
+
+	- [Read a cluster summary](cloud-api.html#get-information-about-a-specific-cluster).
+	- [Manage Customer-Managed Encryption Keys (CMEK) for Dedicated Clusters](managing-cmek.html)
+	- [Export a cluster's logs](export-logs.html).
+	- [Export a cluster's metrics](export-metrics.html).
+	- [View and configure a cluster's Egress Rules](egress-perimeter-controls.html).
+	- [Configure the export of metrics to DataDog or AWS CloudWatch](export-metrics.html).
+
+This role can be considered a more restricted alternative to [Cluster Administrator](#cluster-administrator), as it grants all of the permissions of that role, except that it does **not** allow users to:
+
+- Managing cluster-scoped roles on organization users.
```

Comment:
It sounds from the sentence prior to the list here, that you should drop the gerund, so just have: 

does not allow users to....

- Manage cluster-scoped roles..
- Manage SQL users...
- Create or delete...